<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTViewWidgets</class>
 <widget class="QWidget" name="CTViewWidgets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>779</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TopStateBar</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout_sliceView"/>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_picker"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSlider" name="verticalSlider">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QGridLayout" name="gridLayout_3dView"/>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_resultview"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="verticalLayout_toolbar"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_CheckResult"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
